js-packages/@fortawesome/fontawesome-free/scss/_larger.scss

```diff
@@ -2,9 +2,11 @@
 // -------------------------
 
 // makes the font 33% larger relative to the icon container
+@use "sass:math";
+
 .#{$fa-css-prefix}-lg {
-  font-size: (4em / 3);
-  line-height: (3em / 4);
+  font-size: math.div(4em, 3);
+  line-height: (3em * 0.25);
```

Comment:
This should be written without a leading zero `line-height: (3em * .25);`

Reply:
@MisatoTremor Feedback applied 👍